hotelontouch/public/navlogo.svg

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
if you have not used this file please remove it

Reply:
Just wait for 10 minutes i am trying to rebase.

Reply:
Please review this.